examples/tutorials/plot_timeseries_example.py

```diff
@@ -41,8 +43,8 @@
 # class to download files that arnt in the sample data.
 goes_lc_1 = lc.GOESLightCurve.create(TimeRange('2012/06/01', '2012/06/02'))
 goes_lc_2 = lc.GOESLightCurve.create(TimeRange('2012/06/02', '2012/06/03'))
-filepath_1 = 'C:\\Users\\alex_\\sunpy\\data\\go1520120601.fits' # ToDo: get this path from the LC DL.
-filepath_2 = 'C:\\Users\\alex_\\sunpy\\data\\go1520120602.fits'
+filepath_1 = sunpy.config.get('downloads', 'download_dir') + '\\go1520120601.fits'

```

Comment:
you should use `os.path.join(sunpy.config.get('downloads', 'download_dir'), 'go1520120601.fits')` as it's platform independant.